src/preferences/AllPreferences.js

```diff
@@ -31,20 +31,53 @@
  * PreferencesManager.on("change", AllPreferences.EMMET, preferenceChanged);
    preferenceChanged();
  * ```
+ * NB: JSDOC/Comments marked with * are required for adding the option to Menu
  */
 
-   define(function (require, exports, module) {
+define(function (require, exports, module) {
     const PreferencesManager = require("preferences/PreferencesManager");
     const Strings = require("strings");
+    const Menus = require("command/Menus");
+    const AppInit = require("utils/AppInit");
+    const CommandManager = require("command/CommandManager");
+    const Commands = require("command/Commands");
+
+    // * list of all the commands
+    const EMMET_COMMAND_ID = "edit.emmet";
+    const EMMET_COMMAND_REGISTER = CommandManager.register(Strings.CMD_TOGGLE_EMMET, EMMET_COMMAND_ID, toggleEmmet);
+
+
 
     // list of all the preferences
     const PREFERENCES_LIST = {
```

Comment:
This is redundant. Just use preference_Emmet directly as there is only one pref.